pkg/kubelet/dockertools/manager.go

```diff
@@ -588,11 +588,13 @@ func (dm *DockerManager) runContainer(
 			Hostname:     containerHostname,
 			Image:        container.Image,
 			// Memory and CPU are set here for older versions of Docker (pre-1.6).
-			Memory:     memoryLimit,
-			MemorySwap: -1,
-			CPUShares:  cpuShares,
-			WorkingDir: container.WorkingDir,
-			Labels:     labels,
+			Memory:      memoryLimit,
+			MemorySwap:  -1,
+			CPUShares:   cpuShares,
+			WorkingDir:  container.WorkingDir,
+			Labels:      labels,
+			AttachStdin: container.Stdin,
```

Comment:
Yes, it is a best effort at this moment. Even for today's rkt support, there are several features are missing / underdevelopment, for example, kubectl logs is missing from rkt. 

I am thinking about letting kubelet maintain a small list of feature set for container runtime version, and publish it as NodeStatus?